sklearn/model_selection/_search.py

```diff
@@ -1165,11 +1177,13 @@ class RandomizedSearchCV(BaseSearchCV):
         Either estimator needs to provide a ``score`` function,
         or ``scoring`` must be passed.
 
-    param_distributions : dict
+    param_distributions : dict or list of dicts
         Dictionary with parameters names (string) as keys and distributions
         or lists of parameters to try. Distributions must provide a ``rvs``
         method for sampling (such as those from scipy.stats.distributions).
         If a list is given, it is sampled uniformly.
+        If a list of dicts is given, for each parameter, one of the dicts
```

Comment:
This is slightly unclear. It looks like, first dicts are sampled uniformly, then the parameters are sampled based on that dict.